stratis_cli_cert.py

```diff
@@ -204,6 +204,12 @@ def unittest_command(
         else:
             self.assertNotEqual(stdout, "")
 
+    def test_access_stratis_man_page(self):
```

Comment:
You should do a similar thing for as for stratisd_cert.py here. However, the class heirarchy right now is simpler than in stratisd_cert.py. It's not ready to have a new class added. It'll be necessary to make at least two new classes, similar to the classes that are already defined in stratisd_cert.py.

Reply:
OK, should be all set.